apps/snitch_core/test/data/model/country_test.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Snitch.Data.Model.CountryTest do
+  use ExUnit.Case, async: true
+  use Snitch.DataCase
+  alias Snitch.Data.Model.Country
+  import Snitch.Factory
+
+  setup :countries
+
+  describe "Countries" do
```

Comment:
`describe` is usually used to group tests related to a particular type. 
e.g. It would be better to group tests by `functions` in the model. Here single describe has no purpose these since all the tests are within this.